src/Console/Commands/CrudModelBackpackCommand.php

```diff
@@ -101,45 +102,31 @@ public function handle()
         if (! $this->hasOption('force') || ! $this->option('force')) {
             $this->progressBlock('Adding CrudTrait to the Model');
 
-            $file = $this->files->get($path);
-            $lines = preg_split('/(\r\n)|\r|\n/', $file);
+            $content = Str::of($this->files->get($path));
 
             // check if it already uses CrudTrait
             // if it does, do nothing
-            if (Str::contains($file, $this->crudTrait)) {
+            if (Str::contains($content, $this->crudTrait)) {
```

Comment:
If we just created `$content` which is a `Str` object, that means we can use that `$content` object and fluent string helpers going forward, which are easier to read:

```suggestion
            if ($content->contains($this->crudTrait)) {
```